src/seedu/addressbook/data/address/PostalCode.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.data.address;
+
+/**
+ * Represents a PostalCode in the address.
+ */
+public class PostalCode {
+
+    private String postalCode;
+
+    public String getPostalCode() {
+        return postalCode;
+    }
+
+    public void setPostalCode(String postalCode) {
+        this.postalCode = postalCode;
+    }
+
+    public PostalCode(String postalCode) {
+
+        this.postalCode = postalCode;
```

Comment:
You can try to move the validation logic inside these classes from the main Address class.